moveit_ros/planning/trajectory_execution_manager/cfg/TrajectoryExecutionDynamicReconfigure.cfg

```diff
@@ -7,5 +7,6 @@ gen.add("execution_duration_monitoring", bool_t, 1, "Monitor the execution durat
 gen.add("allowed_execution_duration_scaling", double_t, 2, "Accept durations that take a little more time than specified", 1.1, 1, 10)
 gen.add("allowed_goal_duration_margin", double_t, 3, "Allow more than the expected execution time before triggering a trajectory cancel (applied after scaling)", 0.5, 0.1, 5)
 gen.add("execution_velocity_scaling", double_t, 4, "Multiplicative factor for execution speed", 1, 0.1, 10)
+gen.add("allowed_start_tolerance", double_t, 5, "Allowed joint-value tolerance for validation of trajectory's start point against current robot state", 1e-3, 0);
```

Comment:
Having `8e-5` on UR5, the suggested `1e-3` seems to be a good compromise. It's safe with regard to a 1m lever only "jumping" 1mm in that case. I agree to @v4hn that the value can be made more restrictive if necessary. I wouldn't increase further (e.g. to `1e-2` as suggested in https://github.com/ros-planning/moveit/pull/63/files#r75665102), because that would move by 1cm.